tests/phpunit/integration/Core/Permissions/PermissionsTest.php

```diff
@@ -295,11 +297,14 @@ private function verify_view_shared_dashboard_capability( $author, $contributor
 		// item dismissed to VIEW_SHARED_DASHBOARD.
 		$this->assertFalse( user_can( $author, Permissions::VIEW_SHARED_DASHBOARD ) );
 		$this->assertFalse( user_can( $contributor, Permissions::VIEW_SHARED_DASHBOARD ) );
+		$this->assertFalse( user_can( $contributor, Permissions::VIEW_DASHBOARD ) );
 
 		$contributor_user_options = new User_Options( $this->context, $contributor->ID );
 		$dismissed_items          = new Dismissed_Items( $contributor_user_options );
 		$dismissed_items->add( 'shared_dashboard_splash', 0 );
 		$this->assertTrue( user_can( $contributor, Permissions::VIEW_SHARED_DASHBOARD ) );
+		// User should also be able to access VIEW_DASHBOARD as he has the VIEW_SHARED_DASHBOARD access.
```

Comment:
I think this comment should be tweaked slightly to be in line with our inclusive language guidelines
```suggestion
		// User should also be able to access VIEW_DASHBOARD as they have the VIEW_SHARED_DASHBOARD access.
```